### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    [resources.install_dir]
+
+    [resources.permissions]
+
+    [resources.apt]
+    packages = "mariadb-server, libapr1, libaprutil1, libaprutil1-dbd-sqlite3, libaprutil1-ldap, liblua5.2-0, fonts-dejavu-core, patch, smistrip, unzip, wget, fping, libcap2-bin, libiksemel3, libopenipmi0, libpam-cap, libsnmp-base, libsnmp40, snmptrapd, snmpd, libjs-prototype, jq, libssh-4, php8.2-fpm, php8.2-bcmath"
+    
+    extras.zabbix.repo = "http://repo.zabbix.com/zabbix/5.0/debian bullseye main"
+    extras.zabbix.key = "https://repo.zabbix.com/zabbix-official-repo.key"
+    extras.zabbix.packages = "zabbix-server-mysql, zabbix-frontend-php, zabbix-sql-scripts, zabbix-agent"
```
Comment (alexAubin) [inline manifest.toml:62]:
```suggestion
    extras.zabbix.packages = "zabbix-server-mysql, zabbix-frontend-php, zabbix-agent"
    #extras.zabbix.packages = "zabbix-server-mysql, zabbix-frontend-php, zabbix-sql-scripts, zabbix-agent"
```

**Comment by ericgaspar** [comment]
!testme

**Pull request closed without merging**